ansible/roles_ocp_workloads/ocp4_workload_portworx/tasks/workload.yml

```diff
@@ -45,6 +45,7 @@
 
 - name: Wait for Portworx cluster to to start
   shell:
+    # yamllint disable-line rule:line-length
```

Comment:
Please use `k8s_info` to wait for the cluster to be running.

Example here: https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles_ocp_workloads/ocp4_workload_openshift_data_foundation/tasks/workload.yml#L121